src/App.js

```diff
@@ -1,32 +1,67 @@
-import React from "react";
+import React, { Component } from "react";
 import Hero from "./Hero";
 import Header from "./Header";
 import Gridview from "./Gridview";
 import "./App.css";
-import WrappedMap from "./MapView";
+import mapleleaf from "./mapleleaf.jpg";
```

Comment:
Please remove line #6 
`import mapleleaf from "./mapleleaf.jpg";` from App.js
and also, delete the file as it's 2MB and not needed in the repo.